bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/jsons/JsonPlaylists.java

```diff
@@ -25,7 +25,7 @@
 @NonNullByDefault
 public class JsonPlaylists {
 
-    public @Nullable Map<String, @Nullable PlayList @Nullable []> playlists;
+    public @Nullable Map<String, PlayList @Nullable []> playlists;
```

Comment:
I'm also not sure here, it could be that the json response contains null-values in null-arrays